e2e/ui/api-client.js

```diff
@@ -1,19 +1,41 @@
 import axios from "axios";
+import https from "https";
+import fs from "fs";
 
+export let NOMAD_PROXY_ADDR = process.env.NOMAD_PROXY_ADDR;
 export let NOMAD_ADDR = process.env.NOMAD_ADDR;
 export let NOMAD_TOKEN = process.env.NOMAD_TOKEN;
+export let NOMAD_CLIENT_CERT = process.env.NOMAD_CLIENT_CERT;
+export let NOMAD_CLIENT_KEY = process.env.NOMAD_CLIENT_KEY;
+export let NOMAD_CACERT = process.env.NOMAD_CACERT;
+export let NOMAD_TLS_SERVER_NAME = process.env.NOMAD_TLS_SERVER_NAME;
 
 if (NOMAD_ADDR == undefined || NOMAD_ADDR == "") {
   NOMAD_ADDR = "http://localhost:4646";
 }
 
+if (NOMAD_PROXY_ADDR == undefined || NOMAD_PROXY_ADDR == "") {
+  NOMAD_PROXY_ADDR = NOMAD_ADDR;
+}
+
 export const client = async (
   path,
   { data, method, headers: customHeaders, ...customConfig } = {},
   parameters
 ) => {
   const url = `${NOMAD_ADDR}/v1`.concat(path);
 
+  var httpsAgent = new https.Agent({keepAlive: true});
+
+  if (NOMAD_CLIENT_CERT != "") {
```

Comment:
`question`: Do we want loose or strict equality here? As it stands:

```
undefined != "" // true
null != "" // true
0 != "" // false
```

Reply:
Good catch!